packages/workbox-webpack-plugin/src/generate-sw.js

```diff
@@ -77,7 +77,10 @@ class GenerateSW {
       (compilation.options.output.publicPath || '') + manifestFilename);
 
     // workboxSWImports might be null if importWorkboxFrom is 'disabled'.
+    let workboxSWImport;
     if (workboxSWImports) {
+      // Get the Workbox SW import from the first element in the array.
+      workboxSWImport = workboxSWImports.shift();
```

Comment:
I'm not sure why you are shifting here. To me it feels safer to have all `workboxSWImports` together, then the possible config and then the developers import scripts.